### PR TITLE
update users

### PR DESCRIPTION
**Pull request by anhdtp-1773**
==> T T-8973- Update users
==> Redmine:https://edu-redmine.sun-asterisk.vn/issues/8973
==> Attachment File:
Screenshot from 2019-03-26 10-41-07
Screenshot from 2019-03-26 10-41-31

### CODE REVIEW

**Comment by anhdtp-1773** [comment]
ready

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<% provide(:title, t(".edit_user")) %>
+<% provide(:button_text, t(".save_changes")) %>
+<h1><%= t ".update_your_profile" %></h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for (@user) do |f| %>
```
Comment (huuphuc93) [inline app/views/users/edit.html.erb]: Xóa ()

File: app/views/users/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      <%= render "shared/error_messages", object: f.object %>
+
```
Comment (huuphuc93) [inline app/views/users/edit.html.erb]: xóa dòng thừa đi em

**Comment by anhdtp-1773** [comment]
ready

**Comment by huuphuc93** [comment]
commented
gộp commit rồi ready lại nhé

**Comment by anhdtp-1773** [comment]
ready

**Pull request merged**